hamilton/function_modifiers/adapters.py

```diff
@@ -233,6 +233,7 @@ def get_input_type_key(key: str) -> str:
             namespace=(
                 (namespace, "load_data") if namespace else ("load_data",)
             ),  # We want no namespace in this case
+            originating_functions=(load_data,),
```

Comment:
OK, this `load_data` function will always be a local one. So its not particularly interesting. The spec for `originating_functions` is that its the function's code that generated it. So this will load up a local lambda, and will actually already be assigned (it should be). 

In the case of materializers this is not assigned, so that's why it errors out. So this here will provide really weird behavior in the decorating case.

Reply:
removing.

Reply:
```python
(loader_cls.load_data,)
``` 
would be a better fit